docs/async-api.md

```diff
@@ -42,3 +42,22 @@ This console command has many options to be configured. See all of them by runni
 `vendor/bin/app-sdk asyncapi:add:message -h`
 
 it will print a help page for this command.
+
+
+## Create code from an existing AsyncAPI
+
+The `vendor/bin/app-sdk build:from:asyncapi` reads an existing AsyncAPI file and creates code out of it. This command creates:
+
+- Message Transfer definitions (XML)
+- Adds handler for Messages that are sent to the application
+
+#### Options
+
+- `asyncapi-file`, can be used to run the generator with a specific AsyncAPI file
+- `project-namespace`, can be used to set a specific project namespace (default: Pyz)
+
+After the command was running you need to generate the transfer objects `vendor/bin/console transfer:generate`. After that you can use the generated transfers to pushMessages.
+
```

Comment:
execution of this command can be automated by this command (in 100% of cases the command will be enabled :))

e.g. with Spryk post-commands feature

Reply:
I don't know about such a feature, can you elaborate a bit on it? Where can I find it, how to configure etc?

Reply:
Ah ok, but it does not make much sense in my case to have it like this. I execute several Spryks and especially many to get the properties into the transfer schema.

It would be way more helpful when we could configure to run at the end of all executed Spryks somehow.